bin/lb-ng

```diff
@@ -50,9 +50,9 @@ function runGenerator() {
   // used to have a bug where it prevented the application from exiting.
   // To work around that issue, we are explicitly exiting here.
   //
-  // The exit is deferred to the next tick in order to prevent the Node bug:
-  // https://github.com/joyent/node/issues/3584
-  process.nextTick(function() {
+  // previously used process.nextTick, but in Node 6 is causing
+  // exit to happen before finish writing leading to missing data
+  process.stdout.once('drain', function() {
```

Comment:
what about stderr?

https://nodejs.org/docs/latest-v0.12.x/api/all.html#all_process_exitcode is better to use, but *not* if you need 0.10 support